src/Illuminate/Support/MessageBag.php

```diff
@@ -236,18 +236,13 @@ public function unique($format = null)
      */
     protected function transform($messages, $format, $messageKey)
     {
-        $messages = (array) $messages;
-
-        // We will simply spin through the given messages and transform each one
-        // replacing the :message place holder with the real message allowing
-        // the messages to be easily formatted to each developer's desires.
-        $replace = [':message', ':key'];
-
-        foreach ($messages as &$message) {
-            $message = str_replace($replace, [$message, $messageKey], $format);
-        }
-
-        return $messages;
+        return collect((array) $messages)
```

Comment:
This cast is here since the very first Laravel commit. Taylor was slightly less skilled at this old time ;)